test/ERC721Contract.ts

```diff
@@ -0,0 +1,48 @@
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+describe("ERC721Contract unit tests", function () {
```

Comment:
Let's add here and further comments to with reminder to rename variables and describe name of the test